### PR TITLE
[Documentation] [zos_tso_command] Add REXX exec example

### LINKED ISSUE #1021: [Documentation] [zos_tso_command] Add module examples on how to execute REXX scripts
### Is there an existing issue for this?

- [X] There are no existing issues.

### Documentation issue description

This issue is so that examples of running REXX scripts contained in data sets are demonstrated. This would allow for users to know how to execute a REXX exec, more on this topic in this git issue #427.

### IBM z/OS Ansible core Version

None

### Ansible module

zos_tso_command

### PR DESCRIPTION
**Pull request by rexemin**
##### SUMMARY
Fixes #1021.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
zos_tso_command

### CODE REVIEW

**Review by fernandofloresg — APPROVED** [review]
LGTM

**Review by AndreMarcel99 — APPROVED** [review]

File: plugins/modules/zos_tso_command.py [hunk 1]
```diff
@@ -111,6 +111,10 @@
            - LISTDSD DATASET('HLQ.DATA.SET') ALL GENERIC
       max_rc: 4
 
+- name: Execute TSO command to run explicitly a REXX script from a data set.
```
Comment (ketankelkar) [inline plugins/modules/zos_tso_command.py:114]: Are all the words necessary? It's not wrong or extraneous per se, 
Could simplify to:
> Execute TSO command to run a REXX script from a data set.

Or, to slightly reword what you've got
>name: Execute TSO command to explicitly run a REXX script from a data set.

Swap the order of `explicitly` and `run`

**Review by ketankelkar — CHANGES_REQUESTED** [review]
nothing maaajor, only a few suggestions for wording and an incorrect URL

File: changelogs/fragments/1065-rexx-exec-tso_command.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+minor_changes:
```
[the remark this answers is not in the document]
Comment (rexemin) [reply changelogs/fragments/1065-rexx-exec-tso_command.yml:1]: good point, but I spoke to @fernandofloresg and @AndreMarcel99 and they recommended using `minor_changes` so users would see it on the release notes and be more aware of it, trivial changes don't show up. But I can still change the fragment

File: plugins/modules/zos_tso_command.py (hunk 1 above)
Comment (rexemin) [reply plugins/modules/zos_tso_command.py:114]: There is a distinction between running a REXX script explicitly and implicitly: https://www.ibm.com/docs/en/zos/3.1.0?topic=exec-running
And the example uses a fully qualified data set outside of a system file, so it is running explicitly.

**Review by ketankelkar — APPROVED** [review]
LGTM, thanks for addressing the concerns I raised

**Pull request merged**